masu/processor/ocp/ocp_report_db_cleaner.py

```diff
@@ -0,0 +1,92 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""Removes report data from database."""
+# pylint: skip-file
```

Comment:
why are we skipping pylint?

Reply:
meant to come back to this one.  I need to try generalize `purge_expired_report_data` for both OCP and AWS.  I can do that now.